docs/concepts/copilot/copilot-data-privacy.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Data Copilot data privacy
+id: copilot-data-privacy
+description: The AI assistant data privacy for data Pipelines and models
+sidebar_position: 11
+tags:
+  - concepts
+  - copilot
+  - generativeai
+  - data privacy
+---
+
+The Prophecy team employs top-notch industry practices to safeguard the security of their application and maintain the privacy of customer data. Below are just a few components of our comprehensive security strategy and system structure:
+
+- Prophecy **does not** store or send anything from your data plane to any third-party LLM providers. Instead, Prophecy makes use of rich metadata to construct the knowledge graph. As a result, Data Copilot can interface with LLM providers while maintaining the privacy of the data itself.
+- Prophecy IDE is hosted on secure servers on AWS. All storage systems are encrypted, and all servers are tightly access controlled and audited. Data is encrypted in transit at all times.
```

Comment:
Nit: 
"Data is encrypted in transit at all times."
Should we explain when we are moving data? (not sending to the OpenAI provider)
I don't know the answer to this question.

Reply:
Ack.

I've added a doc task to see if we should add details here later.